diffie-hellman-keys.js

```diff
@@ -68,6 +69,8 @@ module.exports = class DHKeys {
   }
 
   toBFE () {
+    if (this.format === undefined) throw new Error('format is undefined, please use opts.format')
```

Comment:
:fire: this _could_ be a breaking change.

This class is a helper class which makes it easier to work with DiffieHellman keys:
- multiplying them
- deriving them from: feedKeys (signing keys), raw buffers etc
- converting them to: strings, buffers, BFE format (for envelope spec)

The first pass at guessing the `format` was "ok", but now it's making handling keys a bit annoying for me in keyring

Reply:
I think this is unlikely to be a problem.
marking resolved